pySDC/projects/Second_orderSDC/dampedharmonic_oscillator_run_points.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+from pySDC.projects.Second_orderSDC.penningtrap_Simulation import Stability_implementation
+from pySDC.projects.Second_orderSDC.dampedharmonic_oscillator_run_stability import dampedharmonic_oscillator_params
+from tabulate import tabulate
+
+
+def check_stab_points(
+    points, num_nodes_list, max_iter_list, store=False, filename='./data/stab_results.txt'
+):  # pragma: no cover
+    # Initialize simulation parameters based on the damped harmonic oscillator
+    description = dampedharmonic_oscillator_params()
+    quad_type_list = ['GAUSS', 'LOBATTO', 'RADAU-LEFT', 'RADAU-RIGHT']
+
+    # Initialize data as a list of rows
```

Comment:
This comment is misleading. `data` is initialised as an empty list. Please remove.